Chatify/chat/serializers.py

```diff
@@ -0,0 +1,34 @@
+from .models import User
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.ModelSerializer):
+    confirm_password = serializers.CharField(max_length=255, write_only=True)
+
+    class Meta:
+        model = User
+        fields = (
+            "first_name",
+            "last_name",
+            "email",
+            "username",
+            "profile_photo",
+            "mobile_number",
+            "password",
+            "confirm_password",
+        )
+
+    def create(self, validated_data):
+        instance = super().create(validated_data)
+        raw_password = validated_data.get("password")
+        instance.set_password(raw_password)
```

Comment:
Don't writ this code of set password here else you can check if password is exists then we have to set password